docs/shared/reference/project-configuration.md

```diff
@@ -420,6 +420,28 @@ You can also express task dependencies with an object syntax:
 {% /tab %}
 {% /tabs %}
 
+Starting from v19.5.0, wildcards can be used to define dependencies in the `dependsOn` field.
+
+```json
+{
+  "targets": {
+    "test": {
+      "dependsOn": [
+        {
+          "target": "build", // target name
+          "params": "ignore" // "forward" or "ignore", defaults to "ignore"
+        },
+        "build-*", // support for using wildcards in dependsOn, matches: build-css, build-js and also then-build-something-else
```

Comment:
This wouldn't match then-build-something-else, `^{pattern}` means "apply this pattern to targets on my dependencies", not "target name starts with pattern". These are glob patterns, not regex.

Reply:
I don't have a ^symbol in front of the `build-*` pattern though 🤔  unless you are saying that `{pattern}` also means "target starts with pattern"

Reply:
Yeah, kind of. Wildcards are needed to match anything thats not explicit. See: https://www.glob101.com/view/t2Tzx5SUwo6SZkHO3bTi

Reply:
`*build-*` would find all three tasks.

Reply:
Ah! Apologies, I got myself confused 😮 I will update the PR now.

Reply:
Updated: https://github.com/nrwl/nx/pull/27029/commits/90fc87e7cc02f928db1fec1bb3e5c4c12b7bca38